vernac/comHints.ml

```diff
@@ -64,11 +64,11 @@ let project_hint ~poly pri l2r r =
   (info, false, true, Hints.PathAny, Hints.IsGlobRef (Names.GlobRef.ConstRef c))
 
 let warn_deprecated_hint_constr =
-  CWarnings.create ~name:"deprecated-hint-constr" ~category:"deprecated"
+  CWarnings.create ~name:"fragile-hint-constr" ~category:"automation"
```

Comment:
Note that @SkySkimmer recently introduced the "fragile" category, which could also be relevant in this case (so far, it only has one warning).